### DISCUSSION TITLE
Should we remove tentative voting (a.k.a "if need be")?

### DISCUSSION THREAD

**Discussion by lukevella** [lukevella/rallly#1179] [Polls]
I'd really like to remove tentative voting and revert back to simple yes/no answers. I think it makes the voting UX more complicated and makes results harder to interpret. I understand that it can in some instances provide valuable insights but I'm not sure the pros outweigh the cons as there are many downsides to supporting this feature that have a negative impact on UX and hinder development.

I know there are many users who like this feature as it was added by popular request so I would love to hear from you reasons why you believe it should stay.

Please share: What do you use Rallly for? Why does tentative voting help you? Would not having tentative voting stop you from using Rallly?

**Comment by Vadorequest** [comment]
I believe it really depends on the kind of event your are organizing, so I'm doubtful a "one-fit-all" solution exists.

I would rather have it under a feature flag (an option) rather than removing it completely. That won't solve your potential "software" complexity, but that will address the "UX" side of it.

By default, the option could be turned off, while providing the ability to enable it when creating a poll.

**Comment by iamericfletcher** [comment]
If need be is an incredibly important choice. In my line of work, it signals to other assistants that yes this options works, but I will need to move some other meeting that is less important or critical. It is incredibly useful to me, but I'm very certain I speak on behalf of all my assistant colleagues ;)

**Reply by iamericfletcher** [reply]
It's basically saying - while this option technically works, I will need to do some work (potentially a lot of work) moving something else, if possible please try and confirm the meeting for one of the other options. If that's not possible, then OK to confirm and I'll get busy rescheduling / moving whatever else is in the way. 

It's so incredibly important, especially for us assistants who are assisting impossibly busy people. 

As a point of courtesy, I do try and contact the assistants who indicate "if need be" for a meeting I'm scheduling and happen to need the date and time that "if need be" falls on.

**Comment by Vadorequest** [comment]
Another need I identified is the ability to add "options" to each slot.

Typically, ask whether the participant will be available physically or only remotely (e.g: for a meeting)
This way, we could eventually pick a slot where everybody is available physically.

There might be other use-cases like this, which might not directly be related to tentative voting, but still might be interesting to map out to design the system to be flexible to handle those cases in an easy way. (both when configuring them, and when participants are voting)

**Comment by danielwagn3r** [comment]
We use Rallly as staff planning tool for events. We told our team to not use tentative, because it means half-committed which we know is often useless.
However, I clearly see the use for tentative, too. So having configurable options like [Bitpoll](https://bitpoll.de/) have a use case as well.
